store/src/java/com/zimbra/cs/service/mail/ItemAction.java

```diff
@@ -100,15 +101,15 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
 
         Element action = request.getElement(MailConstants.E_ACTION);
         String operation = action.getAttribute(MailConstants.A_OPERATION).toLowerCase();
-
+        boolean nonExistentIdsRequested = action.getAttribute(MailConstants.A_NON_EXISTENT_IDS).equals("1");
```

Comment:
You need to use `Element.getAttributeBool` which handles the fact that the valid values are 1/0/true/false